### ISSUE TITLE
Change in response body from API?

### ISSUE THREAD

**Issue by digmore** [mrtazz/InstapaperLibrary#3]
Hi,

Thanks for sharing your code.

Unless I'm doing something dumb I think the Instapaper simple API has changed. When I attempt an add request it fails with:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/digmore/python/instapaper/env/local/lib/python2.7/site-packages/instapaperlib/instapaperlib.py", line 105, in add_item
    statustxt = self.add_status_codes[int(status)]
ValueError: invalid literal for int() with base 10: '{"bookmark_id": 675979061}'
```

To me it looks like they return a JSON string in the response body instead of a status code now, so the cast fails. This doesn't seem to match their documentation for the simple API.

I'm doing this so far in my project:
```
--- /tmp/instapaperlib.py       2016-01-09 21:39:51.726889676 +0100
+++ env/local/lib/python2.7/site-packages/instapaperlib/instapaperlib.py        2016-01-09 22:14:12.958946872 +0100
@@ -150,7 +150,10 @@
         try:
             request = urllib2.Request(url, headerdata)
             response = urllib2.urlopen(request)
-            status = response.read()
+            if 'jsonp' in params:
+                status = response.read()
+            else:
+                status = response.getcode()
             info = response.info()
             try:
                 headers['location'] = info['Content-Location']
```

Could you please commit a fix for it? Would be great to be able to take it straight from pip.

Thanks!

**Comment by mrtazz** [comment]
Hi!

I haven't used the library in years and also stopped using instapaper. If you are up for creating a pull request I would happily review and merge it. Otherwise I'll try to find some time to see what it would take to implement the new API.

**Comment by mrtazz** [comment]
fixed with #4 and [released to pypi](https://pypi.python.org/pypi/instapaperlib)
